ros/src/computing/perception/localization/packages/ndt_localizer/launch/ndt_mapping.launch

```diff
@@ -1,34 +1,31 @@
 <!-- -->
 <launch>
 
-  <arg name="use_gpu" default="false" />
-  <arg name="use_fast_pcl" default="false" />
-  <arg name="use_openmp" default="false" />
-  <arg name="use_imu" default="false" />
+  <arg name="method_type" default="0" /> <!-- pcl_generic=0, pcl_anh=1, pcl_anh_gpu=2, pcl_openmp=3 -->
   <arg name="use_odom" default="false" />
+  <arg name="use_imu" default="false" />
   <arg name="imu_upside_down" default="false" />
   <arg name="imu_topic" default="/imu_raw" />
+  <arg name="incremental_voxel_update" default="false" />
 
   <!-- rosrun ndt_localizer ndt_mapping  -->
   <node pkg="ndt_localizer" type="queue_counter" name="queue_counter" output="screen"/>
-  <node pkg="ndt_localizer" type="ndt_mapping" name="ndt_mapping" output="screen" unless="$(arg use_openmp)">
-    <param name="use_gpu" value="$(arg use_gpu)" />
-    <param name="use_fast_pcl" value="$(arg use_fast_pcl)" />
-    <param name="use_openmp" value="$(arg use_openmp)" />
+  <node pkg="ndt_localizer" type="ndt_mapping" name="ndt_mapping" output="screen" if="$(eval method_type != 3)">
```

Comment:
The tag `eval` is new in Kinetic and is not supported in Indigo.
http://wiki.ros.org/roslaunch/XML
Can you rewrite in another expression compatible with Indigo and Kinetic?